common/buildings/TSP_hfe_buildings_exodus.orig

```diff
@@ -0,0 +1,137 @@
+#
+
+#TSP_hfe_exodus_planetary_colony_museum = {
+#	base_build_time = 365
+#	base_cap_amount = 1
+#	icon = building_residence
+#
+#	can_build = yes
+#	can_demolish = yes
+#	can_be_ruined = no
+#	can_be_disabled = yes
+#
+#	category = government
+#
+#	potential = {
+#		exists = owner
+#		owner = { is_gestalt = no }
```

Comment:
This is commented out, but still, you can do only one owner switch instead of 3

Reply:
Okay. This whole file was cast aside and it may be brought back in the deep future. I will cross that bridge when I come to it.